presto-docs/src/main/sphinx/sql/describe-output.rst

```diff
@@ -0,0 +1,85 @@
+===============
+DESCRIBE OUTPUT
+===============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    DESCRIBE OUTPUT statement_name
+
+Description
+-----------
+
+Describes the output columns of a prepared statement.  It returns a table
```

Comment:
Returns a table containing metadata for output columns of a prepared statement. For each output column, the column name (or alias), catalog, schema, table, type, type size in bytes, and a boolean indicating if the column is aliased is returned.